src/NumericalAlgorithms/DiscontinuousGalerkin/Actions/InterfaceActionHelpers.hpp

```diff
@@ -0,0 +1,94 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <unordered_map>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Domain/Mesh.hpp"
+#include "Domain/Direction.hpp"
+#include "Domain/Tags.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/FluxCommunicationTypes.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/MortarHelpers.hpp"
+#include "NumericalAlgorithms/DiscontinuousGalerkin/Tags.hpp"
+
+template<typename Metavariables, typename DataBoxType, typename DirectionsTag,
```

Comment:
maybe stick these in a `detail` namespace, noexcept, etc. (all those things not worth doing until we're sure it's what we want :) )

I'm happy with what you're doing here. Seems like a reasonable way to go :+1: